pallets/runtime/src/test/asset_test.rs

```diff
@@ -32,6 +32,20 @@ type AssetError = asset::Error<TestStorage>;
 
 type OffChainSignature = AnySignature;
 
+#[test]
+fn check_the_test_hex() {
```

Comment:
I guess this is just for debug porpoises, right?

Reply:
It is to test whether the function signature calculation is right or not. I keep this because if down the line function signature calculation is changed in the substrate then we will debug using this test function.